src/cache/hybrid-assignment-cache.ts

```diff
@@ -0,0 +1,37 @@
+import { AssignmentCache } from '@eppo/js-client-sdk-common';
+import { AssignmentCacheKey } from '@eppo/js-client-sdk-common/dist/cache/assignment-cache';
+
+export type WriteableAssignmentCache = AssignmentCache & {
```

Comment:
both caches are readabalbe and writeable--I wonder if we should call these "exportable" and "importable" since the main difference is the ability do dump/suck in _all_ entries at once

Reply:
good point, i'll chew on this for a bit